src/lib/tsconfig-umd.json

```diff
@@ -0,0 +1,39 @@
+// TypeScript config file that is used to compile the library. Target environment needs to be
+// ES2015 since the build process will create FESM bundles using rollup.
+{
+  "compilerOptions": {
+    "declaration": true,
+    "stripInternal": false,
+    "experimentalDecorators": true,
+    "noUnusedParameters": true,
+    "strictNullChecks": true,
+    "importHelpers": true,
+    "newLine": "lf",
+    "module": "es2015",
+    "moduleResolution": "node",
+    "outDir": "../../dist/packages/material",
+    "rootDir": ".",
+    "sourceMap": true,
+    "inlineSources": true,
+    "target": "es2015",
+    "lib": ["es2015", "dom"],
+    "skipLibCheck": true,
+    "types": [],
+    "baseUrl": ".",
+    "paths": {
+      "@angular/cdk/*": ["../../dist/packages/cdk/*/public_api"],
+      "@angular/material/*": ["../../dist/packages/material/*/public_api"]
+    }
+  },
+  "files": [
+    "umd.ts",
+    "typings.d.ts"
+  ],
+  "angularCompilerOptions": {
+    "annotateForClosureCompiler": true,
+    "strictMetadataEmit": true,
+    "flatModuleOutFile": "index.js",
+    "flatModuleId": "@angular/material",
+    "skipTemplateCodegen": true
+  }
+}
```

Comment:
Is that file used anywhere? We now have three `tsconfig-XX` files in the `lib/` package.

Reply:
I was just trying something out with this, meant to delete it.